src/Game.js

```diff
@@ -3,6 +3,8 @@ import { parseId } from './utils';
 let round = 0;
 class Game {
   constructor() {
+    this.currentPlayer= "black";
+    let round = 0;
```

Comment:
Zróbmy z tego właściwość obiektu :)
```suggestion
    this.round = 0;
```